lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -96,35 +141,44 @@ def test_es419_acceptance(self):
     def test_unreleased_lang_resolution(self):
         # Regression test; LOC-85
         self.release_languages('fa')
+        self.user_login()
 
         # We've released 'fa', AND we have language files for 'fa-ir' but
         # we want to keep 'fa-ir' as a dark language. Requesting 'fa-ir'
         # in the http request (NOT with the ?preview-lang query param) should
         # receive files for 'fa'
-        response = self.client.get('/', HTTP_ACCEPT_LANGUAGE='fa-ir')
+        response = self.client.get(self.url, HTTP_ACCEPT_LANGUAGE='fa-ir')
```

Comment:
When the user is logged in the root path '/' does not return an HTML document with the get response and the test will fail when looking for the html, lang tag.  This was how the pre-existing tests handled this issue. The get instead uses /dashboard.